targets/mbedos5/mbed-events.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/mbed-events/#a4ba1b08ef90e2b3b6d442696f4d80a8587494e3
```

Comment:
According to https://docs.mbed.com/docs/mbed-os-handbook/en/5.1/dev_tools/cli/, .lib files are not to be hand-managed: "Warning: mbed CLI stores information about libraries and dependencies in reference files that use the .lib extension (like lib_name.lib). While these files are human-readable, we strongly advise that you don’t edit these manually - let mbed CLI manage them instead." Do we need to check this file in the repository, or should they better be added to .gitignore?